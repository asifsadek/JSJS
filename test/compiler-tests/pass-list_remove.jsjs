List.print_list(List.remove([1,2,4,5], 2));

print("");
List.print_list(List.remove([1,2,4,5], 0));

print("");
List.print_list(List.remove([1,2,4,5], 10));

print("");
List.print_list(List.remove([1,2,4,5], -1));
