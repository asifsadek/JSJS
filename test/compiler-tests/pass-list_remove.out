1
2
5

2
4
5

1
2
4
5

1
2
4
5
